doc/cupertino-getting-started.md

```diff
@@ -6,34 +6,126 @@
+					<!-- Load Uno.Cupertino resources -->
+					<CupertinoColors xmlns="using:Uno.Cupertino" />
+					<CupertinoFonts xmlns="using:Uno.Cupertino" />
+					<CupertinoResources xmlns="using:Uno.Cupertino" />
+
+					<!-- Load custom application resources -->
+					<!-- ... -->
+
+				</ResourceDictionary.MergedDictionaries>
+			</ResourceDictionary>
+		</Application.Resources>
+	</Application>
+	```
 
 ## Customization
+
 ### Customize Color Palette
-The colors used in the Cupertino styles are part of the color palette system, which can be customized to suit the theme of your application. Since this is decoupled from the styles, the application theme can be changed, without having to make a copy of every style and edit each of them.
\ No newline at end of file
+1. In the application's **App Code Library** project, add a new Resource Dictionary named `CupertinoColorsOverride.xaml` under `Style\Application`.
+2. Replace the content with:
+	```xml
+	<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+						xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+		<ResourceDictionary.ThemeDictionaries>
+
+			<!-- Light Theme -->
+			<ResourceDictionary x:Key="Light">
+				<!-- Override CupertinoBlueBrush -->
+				<Color x:Key="CupertinoBlueBrush">#6750A4</Color>
+
+				<!-- Add more overrides here -->
+				<!-- ... -->
+			</ResourceDictionary>
+
+			<!-- Dark Theme -->
+			<ResourceDictionary x:Key="Dark">
+				<!-- Override CupertinoBlueBrush -->
+				<Color x:Key="CupertinoBlueBrush">#D0BCFF</Color>
+
+				<!-- Add more overrides here -->
+				<!-- ... -->
+			</ResourceDictionary>
+
+		</ResourceDictionary.ThemeDictionaries>
+	</ResourceDictionary>
+	```
+3. In `AppResources.xaml`, update `<CupertinoColors />` with the override from the previous steps:
+
+	```xml
+	<CupertinoColors xmlns="using:Uno.Cupertino"
+			OverrideSource="ms-appx:///Style/Application/CupertinoColorsOverride.xaml" />
+	```
+
+### Change Default Font
+
+By default, Uno.Cupertino comes pre-packaged with the [SF Pro](https://developer.apple.com/fonts/) `FontFamily` and automatically includes them in your application. Upon installation of the Uno.Cupertino package, you will have a `CupertinoFontFamily` resource available.
+
+If you would like Uno.Cupertino to use a different font, you can override the default `FontFamily` following these steps:
+1. Add the custom font following [this guide](https://platform.uno/docs/articles/features/custom-fonts.html)
+2. Add a new Resource Dictionary named `CupertinoFontsOverride.xaml` to your **App Code  Library** project, under `Style\Application`.
+3. Replace the content with:
+	```xml
+	<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+		<FontFamily x:Key="CupertinoFontFamily">ms-appx:///PATH_TO_MY_CUSTOM_FONT.ttf</FontFamily>
```

Comment:
Most developers will put this in the class library, and the path is not easy to understand at first:

```suggestion
		<FontFamily x:Key="CupertinoFontFamily">ms-appx:///MyClassLibraryName/PATH_TO_MY_CUSTOM_FONT.ttf</FontFamily>
```
We should maybe provide an actual path example, for a given sample app.